nncf/quantization/algorithms/pipeline.py

```diff
@@ -118,6 +118,8 @@ def run_step(
             current_graph = NNCFGraphFactory.create(current_model)
         current_model = pipeline_step[-1].apply(current_model, current_graph, step_statistics)
 
+        step_statistics.remove_statistic_points(algorithm._algorithm_key)
```

Comment:
Don't we need to go through the `pipeline_step[:-1]` in the same manner as above and remove statistics, @andrey-churkin? Would it affect AAQ somehow?

Reply:
Yup, makes sense, pardon. 
Let me do that rn

Reply:
@KodiaqQ just ran the benchmark:
![image](https://github.com/openvinotoolkit/nncf/assets/80326762/a7b2fddf-a9c0-4611-89cd-0206b623a5e8)